client/src/components/app/app.jsx

```diff
@@ -410,7 +410,7 @@ class App extends React.Component {
           <Overview productId={this.state.productId} avgRating={this.state.avgRating} noOfReviews={this.state.noOfReviews}/>
         </OverviewErrorBoundary>
         <RelatedItems />
-        <QAndA productId={this.state.productId}/>
+        {/* <QAndA productId={this.state.productId}/> */}
```

Comment:
FYI I also did this in my PR. Not sure if that will make merging weird or not

Reply:
Ok